app/services/cap_quantity_and_store_changes.rb

```diff
@@ -0,0 +1,50 @@
+# frozen_string_literal: true
+
+class CapQuantityAndStoreChanges
```

Comment:
Very minor: this class doesn't really **store** changes, does it? Could be `CapQuantityAndReturnChanges`?

Reply:
I take it *store* here is a noun though. Perhaps `CapQuantity` alone? We rarely name a class after its return value, do we?

Reply:
Ahh got it, I definitely thought it was a verb. I like just `CapQuantity`, yes 👍